app/assets/javascripts/discourse/app/components/d-editor.js

```diff
@@ -255,8 +264,10 @@ export default Component.extend(TextareaTextManipulation, {
 
     this._textarea = this.element.querySelector("textarea.d-editor-input");
     this._$textarea = $(this._textarea);
+
     this._applyEmojiAutocomplete(this._$textarea);
     this._applyCategoryHashtagAutocomplete(this._$textarea);
+    this._applyExtraAutocompletes();
```

Comment:
I think it is important that we have tests here to catch regressions especially since it is acting as an API.